mula/setup.py

```diff
@@ -1,3 +1,8 @@
 from setuptools import setup, find_packages
 
-setup(name="mula", author="MinVWS", packages=find_packages(exclude="tests"), include_package_data=True)
+setup(
+    name="mula",
+    author="MinVWS",
+    packages=find_packages(exclude="tests"),
+    include_package_data=True,
+)
```

Comment:
There is a lot of unnecessary reformatting noise in this PR that makes code reviewing what actually has changed hard.